projects/storefrontlib/cms-structure/seo/structured-data/json-ld-script.factory.ts

```diff
@@ -12,27 +12,28 @@ import {
   PLATFORM_ID,
   Renderer2,
   RendererFactory2,
-  SecurityContext,
 } from '@angular/core';
-import { DomSanitizer } from '@angular/platform-browser';
 import { WindowRef } from '@spartacus/core';
 import { SeoConfig } from '../config';
 
 @Injectable({
   providedIn: 'root',
 })
 export class JsonLdScriptFactory {
+  protected renderer: Renderer2;
```

Comment:
Nice! :)

Just to be more consistent with other classes, let's init this property inline:

```ts
  protected renderer: Renderer2 = this.rendererFactory.createRenderer(
    null,
    null
  );

  constructor(
    @Inject(PLATFORM_ID) protected platformId: string,
    protected winRef: WindowRef,
    protected rendererFactory: RendererFactory2,
    protected config: SeoConfig
  ) {}
```

effectively, there won't be any difference when the code is compiled down to .js.

Reply:
good idea, thx